Enterprise-CMCS/macpro-mdct-mfp
# Mdct 2760

### Description
State & Territory-Specific Initiatives Instructions page


### Related ticket(s)


MDCT-2760

---
### How to test
go to /wp/state-and-territory-specific-initiatives/instructions see that it matches the spec in figma
https://www.figma.com/file/oOJMUZerbafZefLQQ33GwI/MFP_Playground?type=design&node-id=1966-183769&mode=design&t=gzWyxvcjmEVU2hqN-0

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
